code/modules/surgery/organs/augments_chest.dm

```diff
@@ -207,3 +207,20 @@
 
 	toggle(silent = TRUE)
 	return 0
+
+/obj/item/organ/cyberimp/chest/ventcrawler
+	name = "Ventcrawler implant"
+	desc = "This implant gives its user the ability to ventcrawl at will."
+	icon_state = "chest_implant"
+	implant_color = "#fffb00"
+	syndicate_implant = TRUE
+
+/obj/item/organ/cyberimp/chest/ventcrawler/Insert()
+	. = ..()
+	owner.ventcrawler = VENTCRAWLER_ALWAYS
+
+/obj/item/organ/cyberimp/chest/ventcrawler/Remove(mob/living/carbon/M, special = FALSE)
+	. = ..()
+	M.ventcrawler = VENTCRAWLER_NONE
```

Comment:
You might want to change it to this, just so monkeys don't get vent privileges removed.

```
/obj/item/organ/cyberimp/chest/ventcrawler/Remove(mob/living/carbon/M, special = FALSE)
	. = ..()
	M.ventcrawler = initial(M.ventcrawler)
```